Framework/PythonInterface/mantid/geometry/src/Exports/PointGroup.cpp

```diff
@@ -40,6 +42,14 @@ boost::python::list getEquivalents(PointGroup &self, const object &hkl) {
 Mantid::Kernel::V3D getReflectionFamily(PointGroup &self, const object &hkl) {
   return self.getReflectionFamily(Converters::PyObjectToV3D(hkl)());
 }
+
+std::string __repr__implementation(const PointGroup &self) {
+  std::stringstream ss;
+  ss << "PointGroupFactory.createPointGroup(\"";
```

Comment:
This will get you back the proper `PointGroup`, so that's good for how it's implemented now. But seeing it written down like this I think this should be simplified to `PointGroup(symbol)` (similar for `SpaceGroup`) - the factory stuff should ideally be hidden away from the users - but it's probably for another PR. It has nothing to do with this code, it just caught my eye while looking at this.